pay-api/src/pay_api/services/base_payment_system.py

```diff
@@ -151,13 +151,14 @@ def _release_payment(invoice: Invoice):
 
     @staticmethod
     def _refund_and_create_credit_memo(invoice: InvoiceModel):
+        from pay_api.services import flags  # pylint: disable=import-outside-toplevel
```

Comment:
If I didn't put the import here, ran into circular dependency troubles or flags not having `is_on`